src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.fileReader;
+import core.basesyntax.dao.fileReaderImpl;
+import core.basesyntax.dao.writeReport;
+import core.basesyntax.dao.writeReportImpl;
+import core.basesyntax.model.Account;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.action.ActionHandler;
+import core.basesyntax.service.action.BalanceAction;
+import core.basesyntax.service.action.PurchaseAction;
+import core.basesyntax.service.action.ReturnAction;
+import core.basesyntax.service.action.SupplyAction;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    public static final Map<Account.Operation, ActionHandler> actionHandlerMap = new HashMap<>();
+    private static fileReader fileReader = new fileReaderImpl();
```

Comment:
The variable `fileReader` should be of type `FileReader` (following Java naming conventions) if the class is renamed accordingly.